docker/package/packages.py

```diff
@@ -213,13 +212,13 @@ def mk_node_unit(
         mk_node_unit(suffix=network, env=env, desc=f"Tezos node {network}")
     )
     node_postinst_steps += f"""mkdir -p /var/lib/tezos/node-{network}
-[ ! -f /var/lib/tezos/node-{network}/config.json ] && tezos-node config init --data-dir /var/lib/tezos/node-{network} --network {network_config}
+[ ! -f /var/lib/tezos/node-{network}/config.json ] && octez-node config init --data-dir /var/lib/tezos/node-{network} --network {network_config}
 chown -R tezos:tezos /var/lib/tezos/node-{network}
 
 cat > /usr/bin/tezos-node-{network} <<- 'EOM'
 #! /usr/bin/env bash
 
-TEZOS_NODE_DIR="$(cat $(systemctl show -p FragmentPath tezos-node-{network}.service | cut -d'=' -f2) | grep 'DATA_DIR' | cut -d '=' -f3 | cut -d '"' -f1)" tezos-node "$@"
+TEZOS_NODE_DIR="$(cat $(systemctl show -p FragmentPath tezos-node-{network}.service | cut -d'=' -f2) | grep 'DATA_DIR' | cut -d '=' -f3 | cut -d '"' -f1)" octez-node "$@"
 EOM
 chmod +x /usr/bin/tezos-node-{network}
```

Comment:
These utility scripts should also be renamed to `tezos-octez-{network}` IMO

However, as these are "fake" binaries of sorts, I think we should also add a symlink for these as well